packages/ecs-agent/ecs.service

```diff
@@ -0,0 +1,34 @@
+[Unit]
+Description=Amazon Elastic Container Service - container agent
+Documentation=https://aws.amazon.com/documentation/ecs/
+Requires=docker.service
+After=docker.service configured.target
+
+[Service]
+Type=simple
+Restart=on-failure
+RestartPreventExitStatus=5
+RestartSec=1s
+EnvironmentFile=-/etc/ecs/ecs.config
+Environment=ECS_LOGFILE=/var/log/ecs/ecs-agent.log ECS_LOGLEVEL=debug
+Environment=ECS_CHECKPOINT=true
+ExecStartPre=mkdir -p /var/log/ecs /var/lib/ecs
```

Comment:
I'd prefer to create these via a tmpfiles snippet; we do this for chrony and wicked. That would also simplify relabeling them if we need to.

Reply:
I did not know about [tmpfiles.d](https://fedoraproject.org/wiki/Packaging:Tmpfiles.d) before.  That seems reasonable.

Reply:
Done